crates/solidity/testing/snapshots/cst_output/ContractDefinition/unterminated_body/generated/0.4.11.yml

```diff
@@ -5,7 +5,7 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected Identifier, or MappingKeyword, or FunctionKeyword, or UnsignedFixedType, or SignedFixedType, or UnsignedIntegerType, or SignedIntegerType, or FixedBytesType, or PayableKeyword, or AddressKeyword, or StringKeyword, or BoolKeyword, or ErrorKeyword, or EventKeyword, or TypeKeyword, or EnumKeyword, or StructKeyword, or ModifierKeyword, or FunctionKeyword, or FunctionKeyword, or UsingKeyword, or CloseBrace
+    Error: Expected Identifier, or UnsignedFixedType, or SignedFixedType, or UnsignedIntegerType, or SignedIntegerType, or FixedBytesType, or PayableKeyword, or AddressKeyword, or StringKeyword, or BoolKeyword, or MappingKeyword, or FunctionKeyword, or ErrorKeyword, or EventKeyword, or TypeKeyword, or EnumKeyword, or StructKeyword, or ModifierKeyword, or FunctionKeyword, or FunctionKeyword, or UsingKeyword, or CloseBrace
```

Comment:
This is the same error message, but with keywords rotated.
I need to make sure this list is always alphabetically sorted.